software/nextcloud-memories.yml

```
@@ -6,9 +6,9 @@ licenses:
 platforms:
   - PHP
 tags:
-  - Photo and Video Galleries
+  - Photo Galleries
 source_code_url: https://github.com/pulsejet/memories
 demo_url: https://demo.memories.gallery/apps/memories/
 stargazers_count: 3273
 updated_at: '2024-12-30'
-archived: false
+archived: false
```

Fixes the change on the last line:
```suggestion
archived: false

```

---

Empty line not needed here.